<!DOCTYPE html>
<html>
	<head>
		<title>Code</title>
		<!-- link to main stylesheet -->
		<link rel="stylesheet" type="text/css" href="/css/main.css">
	</head>
	<body>
        <div class="topnav">
            <a>Bernardo P. de Almeida</a>
            <div class="topnav-right">
                <a href="/">Home</a>
                <a href="/about/">About me</a>
                <a href="/publications/">Publications</a>
        		    <a href="/code/">Code</a>
        		    <a href="/tutorials/">Tutorials</a>
        		    <a href="/BernardoAlmeida_CV.pdf">CV</a>
            </div>
        </div>


        <br/>

        <div style="width:70%;margin-left:auto;margin-right:auto;">
        <h1>Publications</h1>
        <h2>Selected</h2>
				<p class="pub" style="margin:0;"><strong>B.P. de Almeida</strong>, C. Schaub, M. Pagani, S. Secchia, E.E.M. Furlong, A. Stark. <a href="https://www.nature.com/articles/s41586-023-06905-9" class="paper">Targeted design of synthetic enhancers for selected tissues in the Drosophila embryo</a>. <em>Nature</em> 2023</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><u><a href="https://github.com/bernardo-de-almeida/DeepSTARR_embryo/" class="link">CODE</a></u></p>
				
        <p class="pub" style="margin:0;">F. Reiter*, <strong>B.P. de Almeida*</strong>, A. Stark. <a href="https://genome.cshlp.org/content/33/3/346" class="paper">Enhancers display constrained sequence flexibility and context-specific modulation of motif function</a>. <em>Genome Research</em> 2023</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><u><a href="https://github.com/bernardo-de-almeida/Variant_STARRseq/" class="link">CODE</a></u></p>
				
        <p class="pub" style="margin:0"><strong>B.P. de Almeida</strong>, F. Reiter, M. Pagani, A. Stark. <a href="https://www.nature.com/articles/s41588-022-01048-5" class="paper">DeepSTARR predicts enhancer activity from DNA sequence and enables the <em>de novo</em> design of synthetic enhancers</a>. <em>Nature Genetics</em> 2022</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><u><a href="https://github.com/bernardo-de-almeida/DeepSTARR/" class="link">CODE</a></u> <u><a href="https://kipoi.org/models/DeepSTARR/" class="link">MODEL</a></u> <u><a href="https://zenodo.org/records/5502060" class="link">DATA</a></u> <u><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183939" class="link">GEO</a></u></p>
        
        <p class="pub" style="margin:0"><strong>B.P. de Almeida</strong>, A.F. Vieira, J. Paredes, M. Bettencourt-Dias, N.L. Barbosa-Morais. <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006832" class="paper">Pan-cancer association of a centrosome amplification gene expression signature with genomic alterations and clinical outcome</a>. <em>PLoS Computational Biology</em> 2019</p>
        <p class="pub" style="margin:0; padding-top:0;"><u><a href="https://github.com/bernardo-de-almeida/PanCancer_CentrosomeAmplification" class="link">CODE</a></u></p>

        <br/>
        
        <h2>Preprints</h2>
        <p class="pub" style="margin:0"><strong>B.P. de Almeida*</strong>, H. Dalla-Torre*, G. Richard, C. Blum, L. Hexemer, M. Gélard, P. Pandey, S. Laurent, A. Laterre, M. Lang, U. Şahin, K. Beguir, T. Pierrot. <a href="https://www.biorxiv.org/content/10.1101/2024.03.14.584712v2" class="paper">SegmentNT: annotating the genome at single-nucleotide resolution with DNA foundation models</a>. <em>bioRxiv</em> 2024</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:0;"><u><a href="https://github.com/instadeepai/nucleotide-transformer" class="link">CODE</a></u> <u><a href="https://huggingface.co/InstaDeepAI/segment_nt" class="link">MODELS</a></u></p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;">Example notebooks are available on <a href="https://colab.research.google.com/github/instadeepai/nucleotide-transformer/blob/main/examples/inference_segment_nt.ipynb" class="link">Google Colab</a></p>


        <p class="pub" style="margin:0">E. Trop*, C.-H. Kao*, M. Polen*, Y. Schiff*, <strong>B.P. de Almeida</strong>, A. Gokaslan, T. Pierrot, V. Kuleshov. <a href="https://openreview.net/pdf?id=M3VlreGcC1" class="paper">Advancing DNA Language Models: The Genomics Long-Range Benchmark</a>. <em>MLGenX ICLR Workshop</em> 2024</p>
        <p class="pub" style="margin-top:0; padding-top:0; margin-bottom:30px;"><u><a href="https://huggingface.co/datasets/InstaDeepAI/genomics-long-range-benchmark" class="link">DATA</a></u></p>
        
        <p class="pub" style="margin:0">J. J. Garau-Luis*, P. Bordes*, L. Gonzalez*, M. Roller, <strong>B.P. de Almeida</strong>, L. Hexemer, C. Blum, S. Laurent, J. Grzegorzewski, M. Lang, T. Pierrot§, G. Richard§. <a href="https://arxiv.org/abs/2406.14150" class="paper">Multi-modal Transfer Learning between Biological Foundation Models</a>. <em>arXiv</em> 2024</p>
        <p class="pub" style="margin-top:0; padding-top:0; margin-bottom:30px;"><u><a href="https://huggingface.co/isoformer-anonymous/Isoformer" class="link">MODEL</a></u></p>
        
        <p class="pub" style="margin:0">V. Loubiere, <strong>B.P. de Almeida</strong>, M. Pagani,  A. Stark. <a href="https://www.biorxiv.org/content/10.1101/2023.10.10.561770v1" class="paper">Developmental and housekeeping transcriptional programs display distinct modes of enhancer-enhancer cooperativity in <em>Drosophila</em></a>. <em>bioRxiv</em> 2023</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><u><a href="https://github.com/vloubiere/git_peSTARRSeq" class="link">CODE</a></u></p>
        
        <p class="pub" style="margin:0">J.M. Xavier, R. Magno, R. Russell, <strong>B.P. de Almeida</strong>, A. Jacinta-Fernandes, A. Duarte, M. Dunning, S. Samarajiwa, M. O’Reilly, C.L. Rocha, N. Rosli, B.A.J. Ponder, A.T. Maia. <a href="https://www.medrxiv.org/content/10.1101/2022.03.08.22271889v1" class="paper">Mapping of cis-regulatory variants by differential allelic expression analysis identifies candidate risk variants and target genes of 27 breast cancer risk loci</a>. <em>medRxiv</em> 2022</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><u><a href="https://github.com/maialab/DaeBreastMicroarrays" class="link">CODE</a></u></p>

        <br/>
        
        <h2>Others</h2>
        <p class="pub" style="margin:0">13. S. Boshar, E. Trop, <strong>B.P. de Almeida</strong>, L. Copoiu, T. Pierrot. <a href="https://academic.oup.com/bioinformatics/article/40/9/btae529/7745814" class="paper">Are genomic language models all you need? Exploring genomic language models on protein downstream tasks</a>. <em>Bioinformatics</em> 2024</p>
        <p class="pub" style="margin-top:0; padding-top:0; margin-bottom:30px"><u><a href="https://github.com/instadeepai/nucleotide-transformer" class="link">CODE</a></u> <u><a href="https://huggingface.co/InstaDeepAI/nucleotide-transformer-v2-50m-3mer-multi-species" class="link">MODEL</a></u> <u><a href="https://huggingface.co/datasets/InstaDeepAI/true-cds-protein-tasks" class="link">BENCHMARK</a></u></p>
        
        <p class="pub" style="margin:0">12. H. Dalla-Torre, L. Gonzalez, J. Mendoza-Revilla, N.L. Carranza, A.H. Grzywaczewski, F. Oteri, C. Dallago, E. Trop, <strong>B.P. de Almeida</strong>, H. Sirelkhatim, G. Richard, M. Skwark, K. Beguir, M. Lopez, T. Pierrot. <a href="https://www.biorxiv.org/content/10.1101/2023.01.11.523679v3" class="paper">The Nucleotide Transformer: Building and Evaluating Robust Foundation Models for Human Genomics</a>. <em>Nat Methods</em> 2024</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:0"><u><a href="https://github.com/instadeepai/nucleotide-transformer" class="link">CODE</a></u> <u><a href="https://huggingface.co/collections/InstaDeepAI/nucleotide-transformer-65099cdde13ff96230f2e592" class="link">MODELS</a></u> <u><a href="https://huggingface.co/collections/InstaDeepAI/nucleotide-transformer-65099cdde13ff96230f2e592" class="link">DATA</a></u> <u><a href="https://huggingface.co/datasets/InstaDeepAI/nucleotide_transformer_downstream_tasks_revised" class="link">BENCHMARK</a></u></p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px">Two example notebooks showing how to finetune any of the models <a href="https://github.com/huggingface/notebooks/blob/main/examples/nucleotide_transformer_dna_sequence_modelling.ipynb" class="link">with regular finetuning</a> and <a href="https://github.com/huggingface/notebooks/blob/main/examples/nucleotide_transformer_dna_sequence_modelling_with_peft.ipynb" class="link">with LoRA</a> on any of the Nucleotide Transformer tasks are also available in HuggingFace example notebooks.</p>
        
        <p class="pub" style="margin:0">11. J. Mendoza-Revilla, E. Trop, L. Gonzalez, M. Roller, H. Dalla-Torre, <strong>B.P. de Almeida</strong>, G. Richard, J. Caton, N.L. Carranza, M. Skwark, A. Laterre,K. Beguir, T. Pierrot, M. Lopez. <a href="https://www.nature.com/articles/s42003-024-06465-2" class="paper">A Foundational Large Language Model for Edible Plant Genomes</a>. <em>Commun Biol</em> 2024</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px"><u><a href="https://github.com/instadeepai/nucleotide-transformer" class="link">CODE</a></u> <u><a href="https://huggingface.co/InstaDeepAI/agro-nucleotide-transformer-1b" class="link">MODELS</a></u> <u><a href="https://huggingface.co/datasets/InstaDeepAI/plant-multi-species-genomes" class="link">DATA</a></u> <u><a href="https://huggingface.co/datasets/InstaDeepAI/plant-genomic-benchmark" class="link">BENCHMARK</a></u></p>
        
        <br/>
        
        <h2>Softwares</h2>
        <p class="pub" style="margin:0"><strong>B.P. de Almeida</strong>*, N. Saraiva-Agostinho*, N.L. Barbosa-Morais. <a href="https://bioconductor.org/packages/release/bioc/html/cTRAP.html" class="paper">cTRAP: Identification of candidate causal perturbations from differential gene expression data</a>. R package</p>
        <p class="pub" style="margin:0; padding-top:0;"><a href="https://github.com/nuno-agostinho/cTRAP" class="link">CODE</a></p>

        <br/>
        <p>* equal contribution, § shared corresponding author</p>
        
        <br/>
        <h2>Other resources</h2>
        <p class="pub" style="margin:0"><strong>Reference compendium of non-redundant TF motifs</strong> - Clustering of 6,502 TF motif models from multiple species (mostly focused on <em>Drosophila</em> and human) by similarity to remove redundancy</p>
        <p class="pub" style="margin:0; padding-top:0;"><a href="https://github.com/bernardo-de-almeida/motif-clustering" class="link">CODE</a></p>

        <br/>
        <br/>
        <p>Check my <a href="https://github.com/bernardo-de-almeida" class="link">GitHub</a> account for more details.</p>

        <br/>
        <br/>

        <footer>
    		<ul>
        		<li><a href="mailto:bernardo.almeida94@gmail.com">E-mail</a></li>
                <li><a href="https://twitter.com/deAlmeida_BP">Twitter</a></li>
                <li><a href="https://github.com/bernardo-de-almeida">GitHub</a></li>
                <li><a href="https://www.researchgate.net/profile/Bernardo-De-Almeida-2">ResearchGate</a></li>
                <li><a href="https://www.linkedin.com/in/bpalmeida/">LinkedIn</a></li>
                <li><a href="https://orcid.org/0000-0002-6084-6775">ORCID</a></li>
			</ul>
		</footer>
	</body>
</html>
