<!DOCTYPE html>
<html>
	<head>
		<title>Code</title>
		<!-- link to main stylesheet -->
		<link rel="stylesheet" type="text/css" href="/css/main.css">
	</head>
	<body>
        <div class="topnav">
            <a>Bernardo P. de Almeida</a>
            <div class="topnav-right">
                <a href="/">Home</a>
                <a href="/about/">About me</a>
                <a href="/publications/">Publications</a>
        		    <a href="/code/">Code</a>
        		    <a href="/tutorials/">Tutorials</a>
        		    <a href="/BernardoAlmeida_CV.pdf">CV</a>
            </div>
        </div>


        <br/>

        <div style="width:70%;margin-left:auto;margin-right:auto;">
        <h2>Publications</h2>
				<p class="pub" style="margin:0;"><strong>B.P. de Almeida</strong>, C. Schaub, M. Pagani, S. Secchia, E.E.M. Furlong, A. Stark. <a href="https://www.nature.com/articles/s41586-023-06905-9" class="paper">Targeted design of synthetic enhancers for selected tissues in the Drosophila embryo</a>. <em>Nature</em> 2023</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><a href="https://github.com/bernardo-de-almeida/DeepSTARR_embryo/" class="link">CODE</a></p>
				
        <p class="pub" style="margin:0;">F. Reiter*, <strong>B.P. de Almeida*</strong>, A. Stark. <a href="https://genome.cshlp.org/content/33/3/346" class="paper">Enhancers display constrained sequence flexibility and context-specific modulation of motif function</a>. <em>Genome Research</em> 2023</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><a href="https://github.com/bernardo-de-almeida/Variant_STARRseq/" class="link">CODE</a>   <u><a href="https://www.biorxiv.org/content/10.1101/2022.08.31.506061v1" class="link">Preprint</a></u></p>
				
        <p class="pub" style="margin:0"><strong>B.P. de Almeida</strong>, F. Reiter, M. Pagani, A. Stark. <a href="https://www.nature.com/articles/s41588-022-01048-5" class="paper">DeepSTARR predicts enhancer activity from DNA sequence and enables the <em>de novo</em> design of synthetic enhancers</a>. <em>Nature Genetics</em> 2022</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><u><a href="https://github.com/bernardo-de-almeida/DeepSTARR/" class="link">CODE</a></u>   <u><a href="https://www.biorxiv.org/content/10.1101/2021.10.05.463203v1" class="link">Preprint</a></u></p>
        
        <p class="pub" style="margin:0"><strong>B.P. de Almeida</strong>, A.F. Vieira, J. Paredes, M. Bettencourt-Dias, N.L. Barbosa-Morais. <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006832" class="paper">Pan-cancer association of a centrosome amplification gene expression signature with genomic alterations and clinical outcome</a>. <em>PLoS Computational Biology</em> 2019</p>
        <p class="pub" style="margin:0; padding-top:0;"><a href="https://github.com/bernardo-de-almeida/PanCancer_CentrosomeAmplification" class="link">CODE</a></p>

        <br/>
        <br/>
        
        <h2>Preprints</h2>
        <p class="pub" style="margin:0"><strong>B.P. de Almeida*</strong>, H. Dalla-Torre*, G. Richard, C. Blum, L. Hexemer, M. Gélard, P. Pandey, S. Laurent, A. Laterre, M. Lang, U. Şahin, K. Beguir, T. Pierrot. <a href="https://www.biorxiv.org/content/10.1101/2024.03.14.584712v1" class="paper">SegmentNT: annotating the genome at single-nucleotide resolution with DNA foundation models</a>. <em>bioRxiv</em> 2024</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><u><a href="https://github.com/instadeepai/nucleotide-transformer" class="link">CODE</a></u></p>
        
        <p class="pub" style="margin:0">J. Mendoza-Revilla, E. Trop, L. Gonzalez, M. Roller, H. Dalla-Torre, <strong>B.P. de Almeida</strong>, G. Richard, J. Caton, N.L. Carranza, M. Skwark, A. Laterre,K. Beguir, T. Pierrot, M. Lopez. <a href="https://www.biorxiv.org/content/10.1101/2023.10.24.563624v1" class="paper">A Foundational Large Language Model for Edible Plant Genomes</a>. <em>bioRxiv</em> 2023</p>
        <p class="pub" style="margin:0; padding-top:0; margin-bottom:30px;"><u><a href="https://github.com/instadeepai/nucleotide-transformer" class="link">CODE</a></u></p>
        
        <p class="pub" style="margin:0">H. Dalla-Torre, L. Gonzalez, J. Mendoza-Revilla, N.L. Carranza, A.H. Grzywaczewski, F. Oteri, C. Dallago, E. Trop, <strong>B.P. de Almeida</strong>, H. Sirelkhatim, G. Richard, M. Skwark, K. Beguir, M. Lopez, T. Pierrot. <a href="https://www.biorxiv.org/content/10.1101/2023.01.11.523679v3" class="paper">The Nucleotide Transformer: Building and Evaluating Robust Foundation Models for Human Genomics</a>. <em>bioRxiv</em> 2023</p>
        <p class="pub" style="margin:0; padding-top:0"><u><a href="https://github.com/instadeepai/nucleotide-transformer" class="link">CODE</a></u></p>

        <br/>
        <br/>
        <h2>Softwares</h2>
        <p class="pub" style="margin:0"><strong>B.P. de Almeida</strong>*, N. Saraiva-Agostinho*, N.L. Barbosa-Morais. <a href="https://bioconductor.org/packages/release/bioc/html/cTRAP.html" class="paper">cTRAP: Identification of candidate causal perturbations from differential gene expression data</a>. R package</p>
        <p class="pub" style="margin:0; padding-top:0;"><a href="https://github.com/nuno-agostinho/cTRAP" class="link">CODE</a></p>

        <br/>
        <p>* equal contribution § shared corresponding author</p>
        <br/>
        <br/>
        <h2>Others</h2>
        <p class="pub" style="margin:0"><strong>Reference compendium of non-redundant TF motifs</strong> - Clustering of 6,502 TF motif models from multiple species (mostly focused on <em>Drosophila</em> and human) by similarity to remove redundancy</p>
        <p class="pub" style="margin:0; padding-top:0;"><a href="https://github.com/bernardo-de-almeida/motif-clustering" class="link">CODE</a></p>

        <br/>
        <br/>
        <p>Check my <a href="https://github.com/bernardo-de-almeida" class="link">GitHub</a> account for more details.</p>

        <br/>
        <br/>

        <footer>
    		<ul>
        		<li><a href="mailto:bernardo.almeida94@gmail.com">E-mail</a></li>
                <li><a href="https://twitter.com/deAlmeida_BP">Twitter</a></li>
                <li><a href="https://github.com/bernardo-de-almeida">GitHub</a></li>
                <li><a href="https://www.researchgate.net/profile/Bernardo-De-Almeida-2">ResearchGate</a></li>
                <li><a href="https://www.linkedin.com/in/bpalmeida/">LinkedIn</a></li>
                <li><a href="https://orcid.org/0000-0002-6084-6775">ORCID</a></li>
			</ul>
		</footer>
	</body>
</html>
